.github/workflows/beta-build.yml

```diff
@@ -3,46 +3,61 @@ on:
   push:
     branches:
       - beta
+    # only run on changes to these files
+    paths:
+      - "**.js"
+      - "**.jsx"
```

Comment:
Blocks can have `.scss` files which we would want to pick up on if the only change was to CSS

Reply:
ah good point, will fix

Reply:
updated